datafusion/physical-plan/src/joins/nested_loop_join.rs

```diff
@@ -191,18 +190,6 @@ impl ExecutionPlan for NestedLoopJoinExec {
         distribution_from_join_type(&self.join_type)
     }
 
-    fn equivalence_properties(&self) -> EquivalenceProperties {
```

Comment:
Shouldn't this still have `join_equivalent_properties` rather than being removed?

Reply:
Hmm, seems like it. We will discuss tomorrow and update if necessary.

Reply:
> Shouldn't this still have `join_equivalent_properties` rather than being removed?

Thanks @alamb, for pointing this out. I have implemented `join_equivalent_properties` for ` nest_loop_join`